src/Classes/ConfigTab.lua

```diff
@@ -439,6 +441,14 @@ function ConfigTabClass:BuildModList()
 	self.enemyModList = enemyModList
 	local input = self.input
 	local placeholder = self.placeholder
+	--enemy level handeled here becouse its needed to correctly set boss stats
```

Comment:
--enemy level handled here because it's needed to correctly set boss stats

(fixed spelling)